components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodCLIService.kt

```diff
@@ -85,7 +102,17 @@ class GitpodCLIService : RestService() {
         }
     }
 
+    private tailrec suspend fun getControllerClientIdAsync(): ClientId? {
```

Comment:
we should follow logic in withClient to get last focused or opened client, why don't we poll like each 1s with it?

Reply:
Updated!